anti-tracking-threat-model.md

```diff
@@ -0,0 +1,196 @@
+# Anti-tracking Threat Model
+
+WebKit and other browsers are trying to reduce websites' ability to track users
```

Comment:
browsers –> browser engines

Reply:
Isn’t this correct as is? For example, Brave shares the same Browser Engine as Edge and Chromium, but represent their own approaches to reduce tracking ability.

Reply:
WebKit is an engine used in multiple browsers and web rendering applications. If the text should indeed refer to browsers, it should be Safari. But web technologies are mostly about engines.